### DISCUSSION TITLE
Hotkey to open ChatGPT even if Xcode is closed.

### DISCUSSION THREAD

**Discussion by shanecowherd** [intitni/CopilotForXcode#203] [Ideas]
I'd love to use this as my primary GPT query system instead of other third party apps.

I'd like to drop MacGPT and just use this ;)

**Comment by intitni** [comment]
There is no plan to add the hotkey yet, but in 0.15.0, you can open the chat from the menu bar item.

**Comment by shanecowherd** [comment]
<img width="613" alt="Screenshot 2023-05-14 at 4 14 30 PM" src="https://github.com/intitni/CopilotForXcode/assets/442439/eaf9c7ea-f948-4d93-a3ae-f5039c4d21c3">

Thank you!!
